src/org/zaproxy/zap/extension/websocket/manualsend/ManualWebSocketSendEditorDialog.java

```diff
@@ -190,12 +204,18 @@ public void unload() {
 	private static final class WebSocketMessagePanel extends JPanel {
 
 		private static final long serialVersionUID = -3335708932021769432L;
-		
+
+		private static final Logger LOGGER = Logger.getLogger(ManualWebSocketSendEditorDialog.class);
+
 		private final HttpPanel messagePanel;
 
-		private WebSocketUiHelper wsUiHelper;
-	
-		public WebSocketMessagePanel(ChannelSortedListModel channelsModel, JToolBar controlToolbar, HttpPanel messagePanel) throws IllegalArgumentException {
+		private JButton btnReopen;
+		private JButton btnReopenEdit;
+		private JPanel reopenPanel;
+        private String newSecWebSocketKey;
+        private WebSocketUiHelper wsUiHelper;
+
+		public WebSocketMessagePanel(ChannelSortedListModel channelsModel, JToolBar controlToolbar, HttpPanel messagePanel, WebSocketPanelSender sender) throws IllegalArgumentException {
```

Comment:
Shouldn't change the signature of public methods might break other's code. Best to have the old method call the new with a default value or placeholder value. The original could be marked deprecated if necessary.